### ISSUE TITLE
Option to force biometrics on credentials check.

### ISSUE THREAD

**Issue by Zanchee** [auth0/Auth0.swift#443]
### Describe the problem you'd like to have solved
Would it possible to provide an option to enable biometrics such that they are checked every time the `credentials` method on `CredentialsManager` is called? 

### Describe the ideal solution
For apps that require more thorough security, like fintech apps, it might be helpful for the SDK to provide the ability to enable biometrics to be confirmed every time the  credentials are attempted to be refreshed.

## Alternatives and current work-arounds
I suppose one could disable biometrics in  the  Auth0 SDK and simply manage the biometrics themselves, but I figured I'd ask. 😄

**Comment by Widcket** [comment]
Hi @Zanchee, this is already implemented in the SDK. Please check out [this QuickStart](https://auth0.com/docs/quickstart/native/ios-swift/08-touch-id-authentication).

**Comment by Zanchee** [comment]
I have observed that the Face ID verified if the application is a fresh launch or the access token has expired.

**Comment by Zanchee** [comment]
The ask here would be able to use the Auth0 SDK to validate biometrics again after some short amount of time.

For example... Think of a banking app.

The user launches the app and Face ID is validated.
The user then backgrounds the app and then comes back ~5 mins later.
Most banking apps would re-check the users Face ID before allowing them back into the application.

**Comment by Widcket** [comment]
@Zanchee you can. Auth0.swift uses [SimpleKeychain](https://github.com/auth0/SimpleKeychain) to store the credentials. SimpleKeychain [supports reusing](https://github.com/auth0/SimpleKeychain/pull/74) the [LAContext](https://developer.apple.com/documentation/localauthentication/lacontext), and you can pass [your custom instance](https://github.com/auth0/Auth0.swift/blob/master/Auth0/CredentialsManager.swift#L46) of SimpleKeychain to the CredentialsManager. So you'd need to set up an instance with a custom LAContext, and set up the CredentialsManager with it.

**Comment by Zanchee** [comment]
Thank for for the links to the PR. Very helpful!

**Comment by Zanchee** [comment]
Investigating this more... It seems that my re-evaluation of biometrics was suppressed by the system because I was re-using the same credentials manager. You are correct that `evaluatePolicy` got called every time. However, if `evaluatePolicy` is called again on a `LAContext` that was already evaluated, it returns true without prompting the user for biometrics. So moral of the story is that I should not be storing the `CredentialsManager` as a property on my authentication manager, and instead by creating a new instance of `CredentialsManager` each time.

Hopefully this helps anyone else who stumbles into the same issue. 

TLDR: If you want the biometrics every time, you must re-create your `CredentialsManager` for each call to `credentials(...)`
